### PR TITLE
SCMOD-7585: Documentation updated for default values

### PR DESCRIPTION
**Pull request by thilagasanthosh**
https://portal.digitalsafe.net/browse/SCMOD-7585

### CODE REVIEW

File: caf-audit-service-container/README.md [hunk 1]
```diff
@@ -7,19 +7,25 @@ This is a docker container for the [CAF Audit Web Service](https://github.com/CA
 #### Environment Variables
 
 ##### CAF\_ELASTIC\_HOST\_AND\_PORT
-Comma separated list of Elasticsearch HOST:PORT value pairs. e.g. 192.168.56.10:9200,192.168.56.20:9200
+Comma separated list of Elasticsearch HOST:PORT value pairs. e.g. 192.168.56.10:9200,192.168.56.20:9200. Default value is elastic search host value constructed from alternate configurations CAF_ELASTIC_HOST_VALUES and CAF_ELASTIC_PORT_VALUE.
+
+##### CAF\_ELASTIC\_HOST\_VALUES
+This is the alternative configuration with comma seperated list of Elasticsearch host names. eg. localhost Default value: localhost.
```
Comment (thilagasanthosh) [inline caf-audit-service-container/README.md]: @dermot-hardy , Could you confirm this default value for hostname?

File: caf-audit-service-container/README.md [hunk 2]
```diff
@@ -7,19 +7,25 @@ This is a docker container for the [CAF Audit Web Service](https://github.com/CA
...
+
+##### CAF\_ELASTIC\_PORT\_VALUE
+This is the alternative configuration for REST port of the Elasticsearch server listens on. e.g 9200. Default value: 9200.
 
 ##### CAF\_ELASTIC\_CLUSTER\_NAME
-Name of the Elasticsearch cluster. e.g. docker-cluster
+Name of the Elasticsearch cluster. e.g. docker-cluster. Default value: elasticsearch-cluster.
```
Comment (thilagasanthosh) [inline caf-audit-service-container/README.md]: Could you confirm this default value for the cluster name?

File: caf-audit-service-container/README.md (hunk 1 above)
Comment (andyreidz) [reply caf-audit-service-container/README.md]: @thilagasanthosh Start the container and confirm these values before sending for review.

File: caf-audit-service-container/README.md (hunk 2 above)
Comment (andyreidz) [reply caf-audit-service-container/README.md]: @thilagasanthosh Start the container and confirm these values before sending for review.

File: caf-audit-service-container/README.md (hunk 1 above)
Comment (thilagasanthosh) [reply caf-audit-service-container/README.md]: No default value for the CAF_ELASTIC_HOST_VALUES and CAF_ELASTIC_PORT_VALUE. If there is no elastic search host configured, AuditConfigurationException exception will be thrown from code.

**Review by gaana-vishwnath — APPROVED** [review]

**Review by gaana-vishwnath — COMMENTED** [review]
can you kindly mention a note saying if both CAF_ELASTIC_HOST_AND_PORT and CAF_ELASTIC_HOST_VALUES are present, which one takes a precedence over other.

**Comment by thilagasanthosh** [comment]
@gaana-vishwnath ,

CAF_ELASTIC_HOST_AND_PORT going to be removed in future. So only CAF_ELASTIC_HOST_VALUES will take the highest precedence.

Please refer email from Dermot attached in Jira
https://portal.digitalsafe.net/browse/SCMOD-7032

**Comment by dermot-hardy** [comment]
`CAF_ELASTIC_HOST_AND_PORT` is going to be removed in future?  It's the first I've heard of it.  Where have I said that in the e-mail you're referring to?

**Comment by thilagasanthosh** [comment]
@dermot-hardy ,
I mean to this commit
https://github.houston.softwaregrp.net/Verity/deploy/commit/9d8a70ce44d01bdf2947deeeeb5dd1129d4add99
The env variable CAF_ELASTIC_HOST_AND_PORT_VALUES is removed from the marathon .

**Comment by dermot-hardy** [comment]
The fact that the variable is no longer used in Verity does not imply that it will be removed.

**Comment by thilagasanthosh** [comment]
@dermot-hardy ,
Please review the update.

**Comment by andyreidz** [comment]
@gaana-vishwnath is reviewing this now

**Pull request merged**